src/main/java/core/basesyntax/FieldXY.java

```diff
@@ -13,7 +13,30 @@
 public class FieldXY {
 
     public void moveRobot(Robot robot, int toX, int toY) {
-        //your code
+        Direction directionX;
+        if (robot.getX() > toX) {
+            directionX = Direction.LEFT;
+        } else {
+            directionX = Direction.RIGHT;
+        }
```

Comment:
you can cut this code: 
Direction directionX = (robot.getX() < toX) ? Direction.RIGHT : Direction.LEFT;

Reply:
Done